app/src/pages/inside/filtersPage/filterGrid/filterName/filterName.jsx

```diff
@@ -71,12 +70,6 @@ export class FilterName extends Component {
           >
             {Parser(this.getHighlightName(filter.name))}
           </span>
-          {filter.share &&
```

Comment:
Should it be deleted?
I still see this icon in the designs: https://zpl.io/VOo11R8
However, on RP 4 (port 8080) there are only globe icons in this modal. Please clarify with BA/UX designer which icons should be visible on the step 2 of the widget creation modal.

Reply:
I've just discussed this question with the team: these icons should be hidden only on filters page. In other places one of these icons should be visible (depending on owner).
Maybe it's better to introduce a prop which hides both icons.